doc/example-config-nginx.md

```diff
@@ -56,17 +56,18 @@ Put the following part into the `server` context of your Nginx configuration.
 
 To let Spreed WebRTC know that it is running in a subpath, you need to set the
 `basePath` configuration in the `[http]` section of your `server.conf`. Use the
-same value as for the `<Location ...>` in the Apache configuration. So if you
+same value as for the `location .. {` in the Nginx configuration. So if you
 use the example from above, make sure you have `basePath = /webrtc` in your
-Spreed WebRTC `server.conf` and restart Spreed WebRTC after the change. Also
-make sure you use the same ports for the `ProxyPass` lines in Apache as they are
-set in the `server.conf` `[http]` `listen` setting.
+Spreed WebRTC `server.conf`. Restart Spreed WebRTC after the change. Also
+make sure to use the same port in the `proxy_pass` line as they are set
+in the `[http]` `listen` setting of `server.conf`.
 
 ## Done, now testing
 
-Make sure to also reload the Apache HTTP server and the WebRTC server should
-become available at https://yourserver/webrtc and is ready to be used from the
-Nextcloud plugin.
+Make sure to reload Nginx. Spreed WebRTC should become available at
+https://yourserver/webrtc and is ready to be used from the Nextcloud plugin.
+Note: Your browser might cache redirects, so we recommend opening the URL in
```

Comment:
We should not recommend that. If you really want to mention a possible problem here then mention that the browser cache should be cleared. I would leave it out as it is unlikely to happen and there is no reason to have accesses the URL before ever.